Libraries/Components/Button.js

```diff
@@ -101,16 +104,9 @@ type ButtonProps = $ReadOnly<{|
 |}>;
 
 /**
- * A basic button component that should render nicely on any platform. Supports
- * a minimal level of customization.
- *
- * <center><img src="img/buttonExample.png"></img></center>
+ * A basic button component that should render nicely on any platform. Supports a minimal level of customization.
  *
- * If this button doesn't look right for your app, you can build your own
- * button using [TouchableOpacity](docs/touchableopacity.html)
- * or [TouchableNativeFeedback](docs/touchablenativefeedback.html).
- * For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js).
- * Or, take a look at the [wide variety of button components built by the community](https://js.coach/react-native?search=button).
+ * If this button doesn't look right for your app, you can build your own button using [TouchableOpacity](touchableopacity) or [TouchableWithoutFeedback](touchablewithoutfeedback). For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js). Or, take a look at the [wide variety of button components built by the community](https://js.coach/?menu%5Bcollections%5D=React%20Native&page=1&query=button).
```

Comment:
I've converted these comments specified in multiple line to single line to make it consistent with the docs. What are your suggestions on using single line or multiline?

Reply:
I think for code readability (and automatic formatters like Prettier), we need to be able to cap line length

My recommendation would be to follow what Markdown does, which is that a single line break is ignored, but a double line break annotates a new paragraph. I'd also try to avoid the leading `*`s at the start of every line, as we'll have to parse those out anyway, an they make editing the comment
much harder as to break things on multiple lines will require manually editing the asterisks.

So, here my preference would be to have the output as:
```js
/**
  A basic button component that should render nicely on any platform. Supports a minimal level of customization.
 
  If this button doesn't look right for your app, you can build your own button using
  [TouchableOpacity](touchableopacity) or [TouchableWithoutFeedback](touchablewithoutfeedback). 
  For inspiration, look at the [source code for this button component](https://github.com/facebook/react-native/blob/master/Libraries/Components/Button.js). 
  Or, take a look at the [wide variety of button components built by the community](https://js.coach/?menu%5Bcollections%5D=React%20Native&page=1&query=button).
 
  Example usage:
 
 ```
  import { Button } from 'react-native';
 ...
 
 <Button
    onPress={onPressLearnMore}
    title="Learn More"
    color="#841584"
    accessibilityLabel="Learn more about this purple button"
 />
 ```
 */
```

Reply:
Cool! I don't like * either, I'll work on your suggestion.